frontend/src/lib/api/user/user.js

```diff
@@ -0,0 +1,6 @@
+import client from "../client";
+
+export function get(id){
+  return client.get('/api/v1/user/'+id);
+}
```

Comment:
1. Pass `accessToken` as second parameters
2. Change it to  client.get(`/api/v1/user/${id}`);